src/components/App/App.js

```diff
@@ -101,39 +100,17 @@ class App extends Component {
     }
   };
 
-  handleOnline = () => {
-    this.setState({
-      isOnline: true
-    });
-  };
-
-  handleOffline = () => {
-    this.setState({isOnline: false});
-  };
-
-  connectionError() {
-    return (
-      (!this.state.isOnline ||
-        this.props.loadStoreListError ||
-        this.props.loadUserListError ||
-        this.props.sendMessageError) &&
-      this.state.loggedIn
-    );
-  }
-
   componentWillUnmount() {
     clearTimeout(this.timer);
     clearInterval(this.interval);
     document.body.removeEventListener('touchstart', this.resetTimeoutTimer);
     document.body.removeEventListener('touchmove', this.resetTimeoutTimer);
```

Comment:
My fault as it was me that added event listeners for mouse move and scroll but they need removing on unmount too.